### PR TITLE
SonicWall SMA docs

### PR DESCRIPTION
**Pull request by vg-svitla**

### CODE REVIEW

File: docs/xdr/features/collect/integrations/network/sonicwall_sma.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+- Must have GMS server or On-Prem Analytics server installed and configured.
+- Have an Address Object Created on the Firewall for SonicWall Analytics system.
+- Have an internal log concentrator (Rsyslog)
+
+### Enable Syslog forwarding for SonicWall SMA
+
+1. Log in to the SonicWall SMA appliance’s management interface
+2. Go to `Log > Settings`
+3. In the Log & Alert levels section, define the severity level of log messages.
+4. In the syslog settings, type the ip address and the port of our log concentrator
```
Comment (squioc) [inline docs/xdr/features/collect/integrations/network/sonicwall_sma.md:28]:
```suggestion
4. In the syslog settings, type the ip address and the port of your log concentrator as Primary syslog server.
```

File: docs/xdr/features/collect/integrations/network/sonicwall_sma.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+uuid: 622999fe-d383-4d41-9f2d-eed5013fe463
+name: SonicWall SMA
+type: intake
+
+## Overview
+
+SonicWall Secure Mobile Access offers secure and seamless remote access to corporate resources, applications, and data, enhancing workforce mobility while maintaining robust security and compliance measures.
+
```
Comment (squioc) [inline docs/xdr/features/collect/integrations/network/sonicwall_sma.md:8]:
```suggestion

!!! warning
    Important note - This format is currently in beta. We highly value your feedback to improve its performance.

```

File: docs/xdr/features/collect/integrations/network/sonicwall_sma.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+{!_shared_content/operations_center/integrations/generated/622999fe-d383-4d41-9f2d-eed5013fe463.md!}
+
+## Configure
+
+This setup guide will show you how to forward your SonicWall SMA logs to Sekoia.io by means of a syslog transport channel.
+
+### Prerequisites
+
+- Must have GMS server or On-Prem Analytics server installed and configured.
+- Have an Address Object Created on the Firewall for SonicWall Analytics system.
```
Comment (squioc) [inline docs/xdr/features/collect/integrations/network/sonicwall_sma.md:20]: Where do you get this information?

File: docs/xdr/features/collect/integrations/network/sonicwall_sma.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+- Have an internal log concentrator (Rsyslog)
+
+### Enable Syslog forwarding for SonicWall SMA
+
+1. Log in to the SonicWall SMA appliance’s management interface
```
Comment (squioc) [inline docs/xdr/features/collect/integrations/network/sonicwall_sma.md:25]: Could you add this screenshot in the documentation?

SonicWall_syslog_interface

**Review by squioc — CHANGES_REQUESTED** [review]

File: docs/xdr/features/collect/integrations/network/sonicwall_sma.md (hunk 3 above)
Comment (vg-svitla) [reply docs/xdr/features/collect/integrations/network/sonicwall_sma.md:23]: Yes my fail. Did not cleanup from `sonicwall fw`. Thank you!

**Review by squioc — APPROVED** [review]

**Pull request merged**